frontend/conf/routes

```diff
@@ -47,6 +47,7 @@ GET            /staff/event-details               controllers.Staff.eventDetails
 # Subscription
 POST           /subscription/update-card          controllers.Subscription.updateCard
 OPTIONS        /subscription/update-card          controllers.Subscription.updateCardPreflight
+GET            /subscription/tickets-left         controllers.Subscription.ticketsLeft
```

Comment:
[URL design nerd warning]

This reads weirdly, maybe `ticket-allocation` or `remaining-tickets`